_includes/community_selector.html

```diff
@@ -0,0 +1,14 @@
+  <select id="" onchange="window.location = '/communities/' + this.value">
+    <option value=""></option>
+
+    <optgroup label="Natural Gas">
+      <option
+        {% if page.url == '/communities/tar' %}selected{% endif %}
```

Comment:
Yes `page.permalink` would be smarter. I was considering creating collections for `communities` and `resource_revenues` pages, but wasn't sure if was worth the extra time at this juncture.

Though, typing that out now is making me realize that it is probably the best time for reconfiguration, especially for the `communities` pages because they could have a lot more permutations, whereas the `resource_revenues` are _more or less_ set, right?